### ISSUE TITLE
Latest Plugin Crash

### ISSUE THREAD

**Issue by ycv005** [pichillilorenzo/flutter_inappwebview#1192]
- [x] I have read the [Get Started - Installation](https://inappwebview.dev/docs/get-started/installation/) section
- [x] I have read and done the [Get Started - Setup Android](https://inappwebview.dev/docs/get-started/setup-android/) section
- [x] I have read and done the [Get Started - Setup iOS](https://inappwebview.dev/docs/get-started/setup-ios/) section
- [x] I have already searched for the same problem

## Environment

| Technology           | Version       |
| -------------------- | ------------- |
| Flutter version      |  3.0.0  |
| Plugin version       |  flutter_inappwebview: ^5.4.3+7 (Hybrid mode enable) |
| Android version      |   8   |
| iOS version          |    NA    |
| Xcode version        |    NA   |

Device information: Nexus 5x

## Description
Here, got a crash report on Firebase Crashlytics @pichillilorenzo 
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void g.a.d.a.k.c(java.lang.String, java.lang.Object)' on a null object reference
       at com.pichillilorenzo.flutter_inappwebview.in_app_browser.InAppBrowserActivity.close(InAppBrowserActivity.java:9)
       at com.pichillilorenzo.flutter_inappwebview.in_app_browser.InAppBrowserActivity.onKeyDown(InAppBrowserActivity.java:49)
       at android.view.KeyEvent.dispatch(KeyEvent.java:2691)
       at androidx.core.view.KeyEventDispatcher.activitySuperDispatchKeyEventPre28(KeyEventDispatcher.java:63)
       at androidx.core.view.KeyEventDispatcher.dispatchKeyEvent(KeyEventDispatcher.java:21)
       at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.java:18)
       at androidx.appcompat.app.AppCompatActivity.dispatchKeyEvent(AppCompatActivity.java:22)
       at androidx.appcompat.view.WindowCallbackWrapper.dispatchKeyEvent(WindowCallbackWrapper.java:2)
       at androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.dispatchKeyEvent(AppCompatDelegateImpl.java:8)
       at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:354)
       at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:4733)
       at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4605)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4200)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4166)
       at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4293)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4174)
       at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4350)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4200)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4166)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4174)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4200)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4166)
       at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4326)
       at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:4487)
       at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:2435)
       at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:1998)
       at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:1989)
       at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:2412)
       at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:141)
       at android.os.MessageQueue.nativePollOnce(MessageQueue.java)
       at android.os.MessageQueue.next(MessageQueue.java:325)
       at android.os.Looper.loop(Looper.java:142)
       at android.app.ActivityThread.main(ActivityThread.java:6494)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Expected behavior:** 

**Current behavior:** 

## Steps to reproduce

1. This
2. Than that
3. Then

**Comment by jhyoon99** [comment]
I am in the same situation.


Fatal Exception: java.lang.NullPointerException
Attempt to invoke virtual method 'void k.a.d.a.k.c(java.lang.String, java.lang.Object)' on a null object reference

com.pichillilorenzo.flutter_inappwebview.in_app_browser.InAppBrowserActivity.close (InAppBrowserActivity.java:9)
com.pichillilorenzo.flutter_inappwebview.in_app_browser.InAppBrowserActivity.onKeyDown (InAppBrowserActivity.java:49)
android.view.KeyEvent.dispatch (KeyEvent.java:2691)
androidx.core.view.KeyEventDispatcher.activitySuperDispatchKeyEventPre28 (KeyEventDispatcher.java:63)
androidx.core.view.KeyEventDispatcher.dispatchKeyEvent (KeyEventDispatcher.java:21)
androidx.core.app.ComponentActivity.dispatchKeyEvent (ComponentActivity.java:18)
androidx.appcompat.app.AppCompatActivity.dispatchKeyEvent (AppCompatActivity.java:22)
androidx.appcompat.view.WindowCallbackWrapper.dispatchKeyEvent (WindowCallbackWrapper.java:2)
androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.dispatchKeyEvent (AppCompatDelegateImpl.java:8)
com.android.internal.policy.DecorView.dispatchKeyEvent (DecorView.java:354)
android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent (ViewRootImpl.java:4733)
android.view.ViewRootImpl$ViewPostImeInputStage.onProcess (ViewRootImpl.java:4605)
android.view.ViewRootImpl$InputStage.deliver (ViewRootImpl.java:4147)
android.view.ViewRootImpl$InputStage.onDeliverToNext (ViewRootImpl.java:4200)
android.view.ViewRootImpl$InputStage.forward (ViewRootImpl.java:4166)
android.view.ViewRootImpl$AsyncInputStage.forward (ViewRootImpl.java:4293)
android.view.ViewRootImpl$InputStage.apply (ViewRootImpl.java:4174)
android.view.ViewRootImpl$AsyncInputStage.apply (ViewRootImpl.java:4350)
android.view.ViewRootImpl$InputStage.deliver (ViewRootImpl.java:4147)
android.view.ViewRootImpl$InputStage.onDeliverToNext (ViewRootImpl.java:4200)
android.view.ViewRootImpl$InputStage.forward (ViewRootImpl.java:4166)
android.view.ViewRootImpl$InputStage.apply (ViewRootImpl.java:4174)
android.view.ViewRootImpl$InputStage.deliver (ViewRootImpl.java:4147)
android.view.ViewRootImpl$InputStage.onDeliverToNext (ViewRootImpl.java:4200)
android.view.ViewRootImpl$InputStage.forward (ViewRootImpl.java:4166)
android.view.ViewRootImpl$AsyncInputStage.forward (ViewRootImpl.java:4326)
android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent (ViewRootImpl.java:4487)
android.view.inputmethod.InputMethodManager$PendingEvent.run (InputMethodManager.java:2435)
android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback (InputMethodManager.java:1998)
android.view.inputmethod.InputMethodManager.finishedInputEvent (InputMethodManager.java:1989)
android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished (InputMethodManager.java:2412)
android.view.InputEventSender.dispatchInputEventFinished (InputEventSender.java:141)
android.os.MessageQueue.nativePollOnce (MessageQueue.java)
android.os.MessageQueue.next (MessageQueue.java:325)
android.os.Looper.loop (Looper.java:142)
android.app.ActivityThread.main (ActivityThread.java:6494)
java.lang.reflect.Method.invoke (Method.java)
com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:438)
com.android.internal.os.ZygoteInit.main (ZygoteInit.java:807)

**Comment by muccy** [comment]
Same here sadly

**Comment by calvin-verticalscope** [comment]
+1 on this. Also, happening on Android 8.1 Nexus 5x.

**Comment by lore-co** [comment]
Same on Android 8.1 Nexus 5x!

**Comment by pichillilorenzo** [comment]
It has been fixed starting from version `5.5.0+3`

**Comment by lore-co** [comment]
> It has been fixed starting from version `5.5.0+3`

My App implements 5.5.0+5 but the crash is still there

**Comment by pichillilorenzo** [comment]
@lore-co probably some users still use an older version of your App. In any case, post the error log you got.

**Comment by lore-co** [comment]
#

> @lore-co probably some users still use an older version of your App. In any case, post the error log you got.

Unluckily this isn't the case since the app was published yesterday :(

Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void k.a.d.a.k.e(k.a.d.a.k$c)' on a null object reference
       at com.pichillilorenzo.flutter_inappwebview.in_app_browser.InAppBrowserActivity.dispose(InAppBrowserActivity.java:3)
       at com.pichillilorenzo.flutter_inappwebview.in_app_browser.InAppBrowserActivity.close(InAppBrowserActivity.java:14)
       at com.pichillilorenzo.flutter_inappwebview.in_app_browser.InAppBrowserActivity.onKeyDown(InAppBrowserActivity.java:49)
       at android.view.KeyEvent.dispatch(KeyEvent.java:2691)
       at androidx.core.view.KeyEventDispatcher.activitySuperDispatchKeyEventPre28(KeyEventDispatcher.java:63)
       at androidx.core.view.KeyEventDispatcher.dispatchKeyEvent(KeyEventDispatcher.java:21)
       at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.java:18)
       at androidx.appcompat.app.AppCompatActivity.dispatchKeyEvent(AppCompatActivity.java:22)
       at androidx.appcompat.view.WindowCallbackWrapper.dispatchKeyEvent(WindowCallbackWrapper.java:2)
       at androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.dispatchKeyEvent(AppCompatDelegateImpl.java:21)
       at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:354)
       at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:4733)
       at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4605)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4200)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4166)
       at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4293)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4174)
       at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4350)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4200)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4166)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4174)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4200)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4166)
       at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4326)
       at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:4487)
       at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:2435)
       at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:1998)
       at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:1989)
       at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:2412)
       at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:141)
       at android.os.MessageQueue.nativePollOnce(MessageQueue.java)
       at android.os.MessageQueue.next(MessageQueue.java:325)
       at android.os.Looper.loop(Looper.java:142)
       at android.app.ActivityThread.main(ActivityThread.java:6494)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)

**Comment by pichillilorenzo** [comment]
@lore-co released new version `5.7.1+2` with the fix

**Comment by lore-co** [comment]
> @lore-co released new version `5.7.1+2` with the fix

Thank you!!!

**Comment by pichillilorenzo** [comment]
@lore-co remember to check the [Requirements for 5.7.0](https://inappwebview.dev/docs/5.x.x/intro#requirements-570) before updating the plugin.
